mpitests/test_probs.py

```diff
@@ -169,57 +163,18 @@ def test_prob_two_wire_nonparam(self, tol, operation, GateWires, Wires, C_DTYPE)
             [0, numQubits - 2, numQubits - 1],
         ],
     )
-    @pytest.mark.parametrize(
-        "Wires",
-        [
-            [0],
-            [1],
-            [0, 1],
-            [0, 2],
-            [0, numQubits - 1],
-            [numQubits - 2, numQubits - 1],
-            range(numQubits),
-        ],
-    )
-    @pytest.mark.parametrize("C_DTYPE", [np.complex128])
     def test_prob_three_wire_nonparam(self, tol, operation, GateWires, Wires, C_DTYPE):
         apply_probs_nonparam(tol, operation, GateWires, Wires, C_DTYPE)
 
     @pytest.mark.parametrize("operation", [qml.PhaseShift, qml.RX, qml.RY, qml.RZ])
     @pytest.mark.parametrize("par", [[0.1], [0.2], [0.3]])
     @pytest.mark.parametrize("GateWires", [0, numQubits - 1])
-    @pytest.mark.parametrize(
```

Comment:
Any reason we are removing the wire parameterisations here and elsewhere?

Reply:
The parametrization was moved to the class top. Now all the test functions have it. :rocket: